compiled_starters/gleam/src/grep.gleam

```diff
@@ -0,0 +1,37 @@
+import argv
+import gleam/erlang
+import gleam/io
+import gleam/string
+
+pub fn main() {
+  // You can use print statements as follows for debugging, they'll be visible when running tests.
+  io.println("Logs from your program will appear here!")
+  // Uncomment this to pass the first stage
+  // let args = argv.load().arguments
+  // let assert Ok(input_line) = erlang.get_line("")
+  // case args {
+  //   ["-E", pattern, ..] -> {
+  //     case match_pattern(input_line, pattern) {
```

Comment:
Can we reduce the number of commented lines so that the diff in the UI looks simpler? 

Python for reference:

<img width="794" alt="Screenshot 2024-05-30 at 16 27 44" src="https://github.com/codecrafters-io/build-your-own-grep/assets/3893573/2d7e3b62-f893-442b-8cd0-e8b6574d491c">